db/migrate/20210422171642_add_indexes_to_comments.rb

```diff
@@ -0,0 +1,15 @@
+class AddIndexesToComments < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    add_index :comments, :commentable_type, algorithm: :concurrently unless index_exists?(:comments, :commentable_type)
+    add_index :comments, :deleted, algorithm: :concurrently unless index_exists?(:comments, :deleted)
+    add_index :comments, :hidden_by_commentable_user, algorithm: :concurrently unless index_exists?(:comments, :hidden_by_commentable_user)
```

Comment:
Now, you may be reading this and asking yourself

```
Why not use the new Rails 6.1 sorcery of `if_not_exists`/`if_exists`?
```

Agreed! I couldn't get it to work 🤣 🙈 . I don't think it works with `algorithm: :concurrently`
![Screen Shot 2021-04-21 at 3 08 02 PM](https://user-images.githubusercontent.com/15987080/115759903-c7dfe000-a36e-11eb-8bfb-57a4e2394583.png)

It messes up the query syntax 😬

Reply:
oh gosh!